scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -1,6 +1,7 @@
 {% extends "layout_bs4.html" %}
 {% from "variants/components.html" import gene_cell, frequency_cell %}
 {% from "utils.html" import comments_table %}
+{% from "variants/utils.html" import cell_rank %}
```

Comment:
😄 nice to unify with macros!

Reply:
Famous last words.. :wink:

Reply:
😅

Reply:
What’s wrong?

Reply:
😊The SV version of that macro unification disabled the link to the SV variant page...

Reply:
Cf #1724...